ppx/ast_literal.ml

```diff
@@ -31,17 +31,18 @@ type t = Longident.t
 
 (* TODO should be renamed in to {!Js.fn} *)
 (* TODO should be moved into {!Js.t} Later *)
```

Comment:
I guess part of this todo is being done here, not sure if it can be removed.

Reply:
I suppose we can remove it.